### PR TITLE
Refine README + docs

### PR DESCRIPTION
**Pull request by juliasilge**
This PR refines the README and docs a bit as we prepare for an announcement, as well as updating URLs.

Let me know if I've bungled any of this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,42 +9,63 @@ experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](h
...
-my_app.run()
+from pins import board_temp
+from vetiver import vetiver_pin_write
+
+model_board = board_temp(versioned = True, allow_pickle_read = True)
+vetiver_pin_write(model_board, v)
 ```
 
-To view more, see [this repo of examples](https://github.com/isabelizimm/vetiverpydemo).
+You can **deploy** your pinned `VetiverModel()` via [FastAPI](https://fastapi.tiangolo.com/).
```
Comment (isabelizimm) [inline README.md:56]:
```suggestion
You can **deploy** your pinned `VetiverModel()` using `VetiverAPI()`, an extension of [FastAPI](https://fastapi.tiangolo.com/).
```

File: README.md [hunk 2]
```diff
@@ -9,42 +9,63 @@ experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](h
...
+python -m pip install git+https://github.com/rstudio/vetiver-python
+```
+
+## Example
+
+A `VetiverModel()` object collects the information needed to store, version, and deploy a trained model. 
 
 ```python
-my_model = VetiverModel(model = linear_reg, ptype_data = train_data)
+from vetiver import mock, VetiverModel, VetiverAPI
```
Comment (isabelizimm) [inline README.md:38]:
```suggestion
from vetiver import mock, VetiverModel
```

File: README.md (hunk 1 above)
Comment (isabelizimm) [reply README.md:56]: Trying to highlight you aren't using FastAPI, but rather the vetiver flavor of it.

**Comment by juliasilge** [comment]
Thank you for those improvements @isabelizimm! 🙌

**Pull request merged**